packages/bpk-component-accordion/src/BpkAccordionItem.module.scss

```
@@ -68,9 +78,25 @@ $bpk-spacing-v2: true;
     &--flipped {
       transform: scaleY(-1);
     }
+
+    &--day {
+      fill: $bpk-canvas-day;
+    }
   }
 
   &__content-container {
     margin: 0;
+
+    &--expanded {
+      padding-bottom: $bpk-spacing-base-v-2;
```

Can you use the function here

```suggestion
      padding-bottom: bpk-spacing-base();
```